### DISCUSSION TITLE
Remove “Update Account Infromation” form after first login

### DISCUSSION THREAD

**Discussion by RuchiSingh18** [keycloak/keycloak#34080] [Q&A]
Hi, I am using keycloak 24.0.5, In that update account information is coming on first time login. I don't want it to be there. Marked all the required field to false in the use profile but still the pop up is coming.
![image](https://github.com/user-attachments/assets/4031cd8a-4e0b-4ee8-81e9-30c16dfc6ca3)

**Comment by codespearhead** [comment]
Please close this discussion [1] in favor of [2], which is already closed.

***

[1] https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#closing-a-discussion
[2] https://github.com/keycloak/keycloak/issues/33758
